App.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import renderer, { ReactTestRendererJSON } from "react-test-renderer";
+
+import App from "./App";
+
+jest.useFakeTimers();
```

Comment:
https://stackoverflow.com/questions/50793885/referenceerror-you-are-trying-to-import-a-file-after-the-jest-environment-has